easybuild/tools/options.py

```diff
@@ -444,8 +447,26 @@ def postprocess(self):
             if token is None:
                 raise EasyBuildError("Failed to obtain required GitHub token for user '%s'", self.options.github_user)
 
+        self._postprocess_external_modules_metadata()
+
         self._postprocess_config()
 
+    def _postprocess_external_modules_metadata(self):
+        """Parse file(s) specifying metadata for external modules."""
+        parsed_external_modules_metadata = ConfigObj()
+        for path in self.options.external_modules_metadata:
```

Comment:
just pass the whole list to configobj in one go? (unless configObj doesn't raise error on missing files)